pages/common/ssh-keyscan.md

```diff
@@ -0,0 +1,15 @@
+# ssh-keyscan
+
+> Get the public ssh keys of remote hosts.
+
+- Retrieve all public ssh keys of a remote host:
+
+`ssh-keyscan {{host}}`
+
+- Retrieve all public ssh keys of a remote host listening on a specific port:
+
+`ssh-keyscan -p {{port}} {{host}}`
+
+- Retrieve certain types of public ssh keys of a remote host:
+
+`ssh-keyscan -t rsa,dsa,ecdsa,ed25519 {{host}}`
```

Comment:
Maybe we could wrap `rsa,dsa,ecdsa,ed25519` in `{{` and `}}`?
```suggestion
`ssh-keyscan -t {{rsa,dsa,ecdsa,ed25519}} {{host}}`
```

Reply:
Done in f4691d7ffa6d48706185ff605b526971e89cf043